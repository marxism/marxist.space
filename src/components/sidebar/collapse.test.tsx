/* eslint-env jest */

import * as React from 'react'
import Collapse from './collapse'
import { create } from 'react-test-renderer'

describe('Collapse', () => {
  test('renders', () => {
    expect(
      create(
        <Collapse title="asdf">
          <span>hello</span>
        </Collapse>
      ).toJSON()
    ).toMatchSnapshot()
  })
})
