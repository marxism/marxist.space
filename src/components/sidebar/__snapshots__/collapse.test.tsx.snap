// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Collapse renders 1`] = `
.c0 {
  cursor: pointer;
  border: 1px solid;
  padding: 16px;
  margin: 4px 0;
}

<div
  className="c0"
  onClick={[Function]}
>
  âŒ„ asdf
</div>
`;
